ipynb/agp_distance_decay.ipynb

```diff
@@ -0,0 +1,279 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This Jupyter notebook recreates the distance-decay analyses for the American Gut Project paper by McDonald et al.\n",
+    "\n",
```

Comment:
Do we need to add that this is a notebook to be run in R?
Should we be generally noting the language of the notebooks we're using?